mlrun/common/constants.py

```diff
@@ -21,3 +21,31 @@
     f"{MLRUN_SERVING_SPEC_MOUNT_PATH}/{MLRUN_SERVING_SPEC_FILENAME}"
 )
 MYSQL_MEDIUMBLOB_SIZE_BYTES = 16 * 1024 * 1024
+MLRUN_INTERNAL_LABELS = [
+    "dask.org/cluster-name",
+    "dask.org/component",
+    "host",
+    "job-type",
+    "kind",
+    "mlrun-auth-keynuclio.io/project-name",
+    "mlrun/class",
+    "mlrun/client_python_version",
+    "mlrun/client_version",
+    "mlrun/function",
+    "mlrun/job",
+    "mlrun/name",
+    "mlrun/owner",
+    "mlrun/owner_domain",
+    "mlrun/project",
+    "mlrun/runner-pod",
+    "mlrun/schedule-name",
+    "mlrun/scrape-metrics",
+    "mlrun/tag",
+    "mlrun/uid",
+    "mlrun/username",
+    "mlrun/username_domain",
+    "owner",
+    "resource_name",
+    "v3io_user",
+    "workflow",
+]
```

Comment:
This is not helpful to understand each label usage and does not link to its usages - thus, prone to errors (such as label changes, etc).

What I do suggest is
- make it a class, where each subclass is the resource name (e.g. "functions", "runs", "artifacts", ...)
- values would be named so it could be reused - DRY

e.g.

class InternalLabels:
  class Runs:
    hostname = "hostname"

and so on.

also. note that this code should be used from mlrun server mostly. because mlrun client should not use/enforce internal labels.

Reply:
About the subclasses, I think we can do it in the following PR